src/components/AppLayout.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { Header } from "./Header";
+import { Footer } from "./Footer";
+import { JokeListPage } from "./JokeListPage";
+
+export function AppLayout() {
+  return (
+    <>
+      <Header />
+      <JokeListPage />
```

Comment:
1. This JokeListPage should be passed via "children" props
2. And JokeListPage should be wrapped with some semantic html tag due to a11y. So it should be in <main>...</main>